src/sentry/similarity/features.py

```diff
@@ -2,115 +2,11 @@
 
 import itertools
 import logging
-import operator
-import struct
-from collections import Sequence
-
-import mmh3
-import six
 
 from sentry.utils.dates import to_timestamp
 
-logger = logging.getLogger('sentry.similarity')
-
-
-FRAME_ITEM_SEPARATOR = b'\x00'
-FRAME_PAIR_SEPARATOR = b'\x01'
-FRAME_SEPARATOR = b'\x02'
-
-FRAME_FUNCTION_KEY = b'\x10'
-FRAME_MODULE_KEY = b'\x11'
-FRAME_FILENAME_KEY = b'\x12'
-FRAME_SIGNATURE_KEY = b'\x13'
-
-
-class InsufficientContext(Exception):
```

Comment:
be4cd255b9f6f2203e2d5f50826c162e0bb880d2, b413adeed3b4614b5b07ebe363be457d96487f45

I don't like it, but it seems like the least bad option I can think of right now.